scout/models/phenotype_term.py

```diff
@@ -1,41 +1,21 @@
-# -*- coding: utf-8 -*-
+from typing import List, Optional
 
+from pydantic import BaseModel, validator
 
-# Hpo terms represents data from the hpo web
-class HpoTerm(dict):
-    """Represents a hpo term
-
-    _id = str, # Same as hpo_id
-    hpo_id = str, # Required
-    hpo_number = int, # Required
-    description = str,
-    genes = list, # List with integers that are hgnc_ids
-    ancestors = list # list with direct ancestors
```

Comment:
Hm, perhaps still keep some of that in a docstring somewhere, and perhaps even like a note/link to the HPO file format definition? It feels kind of obvious to us here and now, but... 😅